api/messages.proto

```diff
@@ -184,7 +184,7 @@ message GatewayTxAcknowledgment {
 }
 
 message GatewayUplinkMessage {
-  UplinkMessage message = 1 [(gogoproto.embed) = true, (validate.rules).message.required = true];
+  UplinkMessage uplink_message = 1 [(validate.rules).message.required = true];
```

Comment:
This is a breaking change. Proto names are part of the API.

Reply:
Oh I see. Ok fixed.